tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-inference-panel.html

```diff
@@ -105,9 +105,14 @@
       <paper-radio-button name="classification">Classification</paper-radio-button>
       <paper-radio-button name="regression">Regression</paper-radio-button>
     </paper-radio-group>
-    <paper-input always-float-label label="Path to examples"
-                 value="{{examplesPath}}">
-    </paper-input>
+    <div class="flex-holder">
+      <paper-input always-float-label label="Path to examples"
+                  value="{{examplesPath}}" class="flex-grow">
+      </paper-input>
+      <paper-checkbox checked="{{sequenceExamples}}" class="checkbox">
+        SequenceExamples
```

Comment:
would you want a whitespace between two words instead of CamelCase?

Reply:
I used camel case since SequenceExample is the proto message name that this checkbox is for.